packages/plugin-link/src/Link.ts

```diff
@@ -55,15 +59,7 @@ export class Link<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
                 commandId: 'unlink',
             },
         ],
-        components: [
-            {
-                id: 'link',
-                async render(): Promise<OwlNode[]> {
-                    return [new OwlNode(LinkComponent, {})];
-                },
-            },
-        ],
-        componentZones: [['link', 'float']],
+        // componentZones: [['link', 'float']],
```

Comment:
I get that this was a poc, but ideally what component to use to complete the required data for a `link` command would be a configuration of the `Link` plugin.